execution/executor-benchmark/src/transaction_generator.rs

```diff
@@ -441,6 +447,63 @@ impl TransactionGenerator {
         }
     }
 
+    /// Generates connected groups of transactions for random pairs of accounts.
+    pub fn gen_independent_grps_of_connected_random_transfer_transactions(
+        &mut self,
+        block_size: usize,
+        num_blocks: usize,
+        independent_tx_grps_in_a_block: usize,
```

Comment:
just rename to `independent_txn_grps`

Reply:
renamed it as 'connected_tx_grps'.. Using 'connected_..' instead of 'independent..' because independent_grp does not clearly indicate whether txns within the same group are independent or independent across the groups